main.go

```diff
@@ -56,6 +58,7 @@ func init() {
 	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
 
 	utilruntime.Must(crunchybridgev1alpha1.AddToScheme(scheme))
+	utilruntime.Must(dbaasredhatcomv1alpha1.AddToScheme(scheme))
```

Comment:
This belongs in `dbaas_support.go`, not `main.go`.

Reply:
moved into dbaas_support.go